README.md

```diff
@@ -186,3 +186,11 @@ First create a `build` folder in the project root. Then proceed as follows.
    ```shell
    conan export-pkg recipes/tket -f --build-folder=build --source-folder=bubble/src
    ```
+
+## API documentation
+
+The `bubble` (C++) API documentation is available
+[here](https://cqcl.github.io/tket/bubble/doc/html/index.html).
```

Comment:
Bubble docs! That's nice. Should we maybe mention that the documentation is very limited for the time being?

Reply:
Done.